packages/asc-ui/src/components/Focus/Focus.tsx

```diff
@@ -1,10 +1,13 @@
 /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
-import React from 'react'
+import React, { HTMLAttributes } from 'react'
+import styled from 'styled-components'
 
 type Props = {
   onKeyDown?: React.EventHandler<React.KeyboardEvent>
 }
 
+const FocusStyle = styled.div<HTMLAttributes<HTMLDivElement>>``
```

Comment:
Do we need to pass this generic type? I think it can be inferred from `styled.div`.

```suggestion
const FocusStyle = styled.div``
```

Reply:
True, good point

Reply:
Pushed